src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -198,6 +198,7 @@ public Pod build() {
             builder.withNodeSelector(nodeSelector);
         }
 
+        builder.withTerminationGracePeriodSeconds(0L);
```

Comment:
Is this really safe?

Reply:
I was wondering as well in case some process *really* need to do cleanup. Probably safer to set it for all tests.

Reply:
> safer to set it for all tests

Do you mean set it _only_ for tests?

Reply:
Yes this is what I meant